lastminute.yml

```diff
@@ -276,15 +278,15 @@
         executable: /bin/bash
       become: false
       register: git_revision
-      delegate_to: 127.0.0.1
+      delegate_to: localhost
       changed_when: false
 
     # TODO to be done in icpc2024
-    - name: Stop and disable apport service
-      ansible.builtin.service:
-          name: apport
-          state: "stopped"
-          enabled: false
+    # - name: Stop and disable apport service
+    #   ansible.builtin.service:
+    #       name: apport
+    #       state: "stopped"
+    #       enabled: false
 
```

Comment:
why was this commented out?

Reply:
seems you are missing the handler?

Reply:
Because there is no apport service on the system, the task fails, and `systemctl status apport` returns nothing. From my test VM:

```
root@backup:~# systemctl status apport
Unit apport.service could not be found.
root@backup:~# dpkg -l | grep apport
ii  python3-apport                                2.20.11-0ubuntu82.3                     all          Python 3 library for Apport crash report handling
```